app/views/estimates/_controlled_work_next_steps.html.slim

```diff
@@ -1,14 +1,18 @@
-- if @check.partner
-  = pdf_friendly_paragraphs(t("estimates.show.controlled_cw_next_steps.intro_paragraphs_partner"), @is_pdf)
-- else
-  = pdf_friendly_paragraphs(t("estimates.show.controlled_cw_next_steps.intro_paragraphs"), @is_pdf)
+= pdf_friendly_p_element(t("estimates.show.controlled_cw_next_steps.intro_paragraph#{'_partner' if @check.partner}"), @is_pdf)
+= yield
```

Comment:
question - so do we `yield` back to `estimates/show.html.slim` & `estimates/print.html.slim` just to show the 'Evidence you might need panel'??

Reply:
Exactly - the show and print screens handle the "Evidence you might need" panel pretty differently. Rather than try to resolve it in here, we `yield` back to them at this point so they can put whatever they want in.